plugins/services/src/js/pages/task-details/TaskIpAddressesRow.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import Application from "#PLUGINS/services/src/js/structs/Application";
+import ConfigurationMapLabel from "#SRC/js/components/ConfigurationMapLabel";
+import ConfigurationMapRow from "#SRC/js/components/ConfigurationMapRow";
+import ConfigurationMapValue from "#SRC/js/components/ConfigurationMapValue";
+import DCOSStore from "#SRC/js/stores/DCOSStore";
+import Pod from "#PLUGINS/services/src/js/structs/Pod";
+
+export default ({ task }) => {
```

Comment:
As far as I know (from @juliangieseke I think) we only use arrow functions when they are necessary. I personally don't like that, I use them in other projects all the time, but I think we should have a convention here

Reply:
...and the convention was _not_ using them at all, and it was me who suggested that :trollface: I will fall back to a regular component. That will allow me to use `null` as a return value. Thank you for 👮